package.json

```diff
@@ -7,6 +7,7 @@
   "funding": "https://github.com/prettier/prettier?sponsor=1",
   "homepage": "https://prettier.io",
   "author": "James Long",
+  "type": "module",
```

Comment:
Does it work to having two separate `index.js` files? Like:

```json
{
  "type": "module", 
  "main": "index.cjs.js",
  "module": "index.esm.js"
}
```

Reply:
This line only make Node.js treat `.js` as  esmodule, nothing about entry.